packages/tools/kadena-cli/src/utils/logger.ts

```diff
@@ -74,7 +76,7 @@ export const defaultTransport: Transport = (record) => {
     [LEVELS.error]: stdErrChalk.red,
     [LEVELS.warning]: stdErrChalk.yellow,
     [LEVELS.output]: stdErrChalk.white,
-    [LEVELS.info]: stdErrChalk.white,
+    [LEVELS.info]: stdErrChalk.gray,
```

Comment:
Info should be the default logging output, why do you prefer it being gray?

Reply:
it was to light on my screen :D